spec/pokedex_spec.rb

```diff
@@ -0,0 +1,349 @@
+require_relative './spec_helper.rb'
+require_relative '../lib/pokedex.rb'
+require 'net/http'
+require 'poke-api-v2'
+
+def expected_data_format
+    pokedex = []
+    
+    pokedex.push({
```

Comment:
no need to use push here.

Create two variables containing your objects.
Then return an array containing the 2 variables.

```
pokemon_one = { 
   id: 1,
   name: "Bulbasaur",
   stats: [ ...all your states],
   types: [... all your types],
   weight: 69
}

pokemon_two = {
   ...similar to above for 2nd one
}

[pokemon_one, pokemon_two]
```

if you want to get really fancy you can create a method to handle formatting the stats and types but for now dont need to worry about that.

Reply:
I keep forgetting about the behaviour of returning in Ruby and this was a helpful reminder so thank you.
I have implemented this change in the latest commit